File: Miscellaneous_Permutation.py
#!/usr/bin/python
from Permutation import Transposition


# Use the combination module/class
goods = ["Eno", "Chidi", "Olu", "Ahmed", "Osas", "Gbeda"]

perm = Transposition()
result = perm.possibleWordPermutations(goods, 9)

# print choices and operation
print("\n", perm.words, " permutation ", perm.r, ":\n")

# print out permutations nicely
for group in result:
    print(group)
    
print("\n\nNumber of ways is ", len(result), ".\n")

File: Miscellaneous_Permutation.cs
using System;
using System.Collections.Generic;

namespace Miscellaneous_CS
{
    class Program
    {
        static void Main(string[] args)
        {
            List<String> goods;
            goods = new List<String>() { "Eno", "Chidi", "Olu", "Ahmed", "Osas", "Gbeda" };
            Permutation perm = new Permutation();
            List<String[]> result = perm.possibleWordPermutations(goods, 5);
            // print choices and operation
            Console.Write("[ ");
            foreach (string choice in perm.words)
            {
                Console.Write(choice + "; ");
            }
            Console.WriteLine("] permutation " + perm.r + ":" + Environment.NewLine);
            // print out permutations nicely
            foreach (String[] set in result)
            {
                foreach (string member in set)
                {
                    Console.Write(member + "; ");
                }
                Console.WriteLine();
            }
            Console.WriteLine(Environment.NewLine + "Number of ways is " + result.Count + ".");
        }
    }
}
